server/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -983,8 +982,6 @@ public synchronized void close() throws IOException {
         toClose.add(injector.getInstance(RemoteClusters.class));
         toClose.add(() -> stopWatch.stop().start("logical_replication_service"));
         toClose.add(injector.getInstance(LogicalReplicationService.class));
-        toClose.add(() -> stopWatch.stop().start("shard_replication_service"));
```

Comment:
as discussed, this is out of the scope of the DROP SUB - causes a lot of test failures